src/main/java/com/amazonaws/kinesisvideo/internal/client/mediasource/MediaSourceSink.java

```diff
@@ -24,6 +25,9 @@ public interface MediaSourceSink {
 
     void onFragmentMetadata(final @Nonnull String metadataName, final @Nonnull String metadataValue, final boolean persistent)
             throws KinesisVideoException;
+    
+    void onEventMetadata(final int event, @Nullable final StreamEventMetadata streamEventMetadata)
```

Comment:
The event is int here but it is byte in the StreamEventMetadata class. Recommend sticking to int in both places for consistency

Reply:
The byte in StreamEventMetadata is for the length of the names/values arrays. The int here is for the event type - these are different things. Both are uniform with their C counterparts (other than being signed).